pages/governance.html

```diff
@@ -9,7 +9,7 @@
 
 <h2 id="dcgc">Governance Committee</h2>
 
-<p>The Data Carpentry Governance Committee provides community leadership for the lesson program. The Committee is responsible for overseeing the strategy of the Data Carpentry project, and advocating for the program, its mission, and its community. The committee acts as a point of contact for Data Carpentry, representing the lesson program to <a href="https://carpentries.org/governance/">The Carpentries Executive Council</a>, the <a href="https://carpentries.org/team/">Core Team</a>, and the wider world. Members of the group volunteer to serve two-year terms.</p>
+<p>The Data Carpentry Governance Committee provides community leadership for the lesson program. The Committee is responsible for overseeing the strategy of the Data Carpentry project, and advocating for the program, its mission, and its community. The committee acts as a point of contact for Data Carpentry, representing the lesson program to <a href="https://carpentries.org/governance/">The Carpentries Governance</a>, the <a href="https://carpentries.org/team/">Core Team</a>, and the wider world. Members of the group volunteer to serve two-year terms.</p>
```

Comment:
```suggestion
<p>The Data Carpentry Governance Committee provides community leadership for the lesson program. The Committee is responsible for overseeing the strategy of the Data Carpentry project, and advocating for the program, its mission, and its community. The committee acts as a point of contact for Data Carpentry, representing the lesson program to <a href="https://carpentries.org/governance/">The Carpentries governance</a>, the <a href="https://carpentries.org/team/">Core Team</a>, and the wider world. Members of the group volunteer to serve two-year terms.</p>
```

Should it be upper or lower case G?

Reply:
Nevermind, we can go ahead and merge this I think.